src/pages/yc-onboarding.tsx

```diff
@@ -129,6 +129,21 @@ export const YCOnboarding = () => {
                                         type: 'string',
                                         name: 'yc_batch',
                                         required: true,
+                                        options: [
+                                            { label: 'Select your batch', value: '' },
+                                            { label: 'Spring 2025 (X25)', value: 'X25' },
```

Comment:
I would make sure to add the next 2 future batches because they do accept earlier these days, like they may have someone for summer as soon as next month.